packages/contracts-bedrock/test/Specs.t.sol

```diff
@@ -799,6 +369,21 @@ contract Specification_Test is CommonTest {
         numEntries[_name]++;
     }
 
+    /// @dev Adds a spec for a function with no auth.
+    function _addSpec(string memory _name, bytes4 _sel, bool _pausable) internal {
+        _addSpec({ _name: _name, _sel: _sel, _auth: Role.NOAUTH, _pausable: _pausable });
+    }
+
+    /// @dev Adds a spec for a function with no pausability.
+    function _addSpec(string memory _name, bytes4 _sel, Role _auth) internal {
+        _addSpec({ _name: _name, _sel: _sel, _auth: _auth, _pausable: false });
+    }
+
+    /// @dev Adds a spec for a function with no auth and no pausability.
+    function _addSpec(string memory _name, bytes4 _sel) internal {
+        _addSpec({ _name: _name, _sel: _sel, _auth: Role.NOAUTH, _pausable: false });
+    }
+
```

Comment:
So, now we have 4 permutations based on the 2 optional arguments. Not bad, but if we added a 3rd optional argument we'd need to have 9 variants. 🤮